### PR TITLE
Passport number is now a required field.

### PR DESCRIPTION
**Pull request by easternbloc**
If we ask for a passport number it is now a required field.

This PR makes it required and updates ALL the tests that assumed it wasn't!

### CODE REVIEW

File: apps/correct-mistakes/translations/src/en/fields.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "label": "Country of nationality"
   },
   "passport": {
-    "label": "Passport number (optional)"
+    "label": "Passport numbe"
```
Comment (timgent) [inline apps/correct-mistakes/translations/src/en/fields.json]: numberrrrrrrr

File: acceptance_tests/features/step_definitions/CollectionFormStep05_.rb [hunk 2]
```diff
@@ -26,7 +27,7 @@
   find_by_id('email')
   find_by_id('no-email')
   page.should have_content('I do not have an email address')
-  page.should have_content('Phone number (optional)')
+  page.should have_content('Phone number')
```
Comment (daniel-ac-martin) [inline acceptance_tests/features/step_definitions/CollectionFormStep05_.rb]: PHONE number. Is this correct?

Comment (easternbloc) [inline acceptance_tests/features/step_definitions/CollectionFormStep05_.rb]: 👓 👀

File: acceptance_tests/features/step_definitions/DeliveryFormStep03.rb [hunk 3]
```diff
@@ -12,8 +12,8 @@
   fill_in('address-postcode', :with => 'SW1P 4DF')
   fill_in('case-id', :with => '1')
   click_button('Continue')
-end                                                                          
-                                                                             
+end
+
```
Comment (daniel-ac-martin) [inline acceptance_tests/features/step_definitions/DeliveryFormStep03.rb]: It's almost as if you don't like pointless white-space!

**Pull request merged**